packages/expo-audio/android/src/main/java/expo/modules/audio/AudioModule.kt

```diff
@@ -63,15 +63,19 @@ class AudioModule : Module() {
       AudioManager.AUDIOFOCUS_LOSS_TRANSIENT -> {
         focusAcquired = false
         players.values.forEach {
-          it.player.pause()
+          runOnMain {
+            it.player.pause()
+          }
         }
       }
 
       AudioManager.AUDIOFOCUS_LOSS_TRANSIENT_CAN_DUCK -> {
         focusAcquired = false
         if (interruptionMode == InterruptionMode.DUCK_OTHERS) {
           players.values.forEach {
-            it.player.volume /= 2f
+            runOnMain {
+              it.player.volume /= 2f
+            }
           }
         }
       }
```

Comment:
when `interruptionMode == InterruptionMode.DO_NOT_MIX` or others shouldn't we pause the player? Or is it paused somewhere else?

Reply:
That's handled in `requestAudioFocus`. We don't prevent mixing in your own app. Interruption mode is only relevant when we are interrupted by another app.

Reply:
Makes sense, thanks!